components/edit_post/edit_post.tsx

```diff
@@ -79,9 +83,9 @@ const {KeyCodes} = Constants;
 const TOP_OFFSET = 0;
 const RIGHT_OFFSET = 10;
 
-const EditPost = ({editingPost, actions, canEditPost, config, ...rest}: Props): JSX.Element | null => {
+const EditPost = ({editingPost, actions, canEditPost, config, teamId, channelId, ...rest}: Props): JSX.Element | null => {
     const [editText, setEditText] = useState<string>(
-        editingPost?.post?.message_source || editingPost?.post?.message || '',
+        rest.draft.message || editingPost?.post?.message_source || editingPost?.post?.message || '',
```

Comment:
Minor nit, but should we destructure `draft` as well instead of having it in `rest`? I'm not actually certain what `rest` does here since it didn't look like it was passed along to a child component when I looked below

Reply:
its now also destructured. I kept `rest` because the remaining values are only used once, or are specific to a function and are destructured there instead.